Clients/.env.example

```
@@ -0,0 +1,11 @@
+# PostHog Analytics Configuration
+# Get your API key from https://app.posthog.com/project/settings
+VITE_POSTHOG_API_KEY=your_posthog_api_key_here
+VITE_POSTHOG_HOST=https://app.posthog.com
+
+# Enable/disable real PostHog tracking
+# Set to 'true' to send events to PostHog, 'false' to use mock client (default: false)
+VITE_REAL_POSTHOG=true
+
+# App Configuration
+VITE_APP_PORT=5173
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)